lib/tasks/correct_renewal_postcodes.rake

```diff
@@ -0,0 +1,17 @@
+desc "Update lettings logs renewal previous postcode and la data to be the same as current postcode/la"
+task correct_renewal_postcodes: :environment do
+  LettingsLog.filter_by_year(2023).where(renewal: 1).where("(ppostcode_full != postcode_full) OR (ppostcode_full IS NULL AND postcode_full IS NOT NULL) OR (postcode_full IS NULL AND ppostcode_full IS NOT NULL)").each do |log|
```

Comment:
I think this is right but just to confirm - there's no way for the la data to differ without one of these being satisfied, right?

Reply:
Ooh, good point, there are a few that differ on la only! Updated the query and tests now. Thanks!